eth_defi/token_tax.py

```diff
@@ -0,0 +1,134 @@
+from email.quoprimime import quote
```

Comment:
Missing module docstring - document what the module is about.

Reply:
Move to `eth_defi/uniswap_v2` folder because the module is specific to Uniswap v2 (swap functions are not available on Uniswap v3).